workspace.go

```diff
@@ -100,6 +107,37 @@ func (w *Workspace) evalCommand(command string) (EvalResult, error) {
 	return res, err
 }
 
+func (w *Workspace) BuildCommit(cmd string) (EvalResult, error) {
+	res, err := w.build(cmd)
+	w.CurrentImage = res.NewImage
+	w.history = append(w.history, res)
+	return res, err
+}
+
+func (w *Workspace) Build(cmd string) (EvalResult, error) {
+	res, err := w.build(cmd)
+	res.Deleted = true
+	w.history = append(w.history, res)
+	return res, err
+}
+
+func (w *Workspace) build(cmd string) (EvalResult, error) {
+	name := uuid.New()
+	c := fmt.Sprintf("FROM %s\n%s", w.CurrentImage, cmd)
+	f, _ := os.Create("." + name)
```

Comment:
We might run into issues on COPY/ADD commands where the pathing gets messed up. If an absolute path is specified, then that's easy to deal with, but relative and knowing what is relative and not gets a bit messy. Maybe we can prefix the builds with `.cyclops_`?